docs/rules/no-class.md

```diff
@@ -0,0 +1,30 @@
+# Avoid use of `class` (no-class)
+
+ECMAScript 6 allows programmers to create `class` to "help" those coming from a OOP language, such as Java or C++, to create constructors, thus multiple instances of an object.
+In the JS community this is considered one of the bad parts (you can google it to read more about it).
```

Comment:
I think you can remove the comment to use google, because you already provided links for further reading at the bottom.